float64 time
franka_msgs/FrankaState franka_state

float64[16] pose
float64[42] zeroJacobian
float64[7] coriolis
float64[7] gravity
float64[49] mass
